MobileSyncExplorerSwift/MobileSyncExplorerSwift/Views/ContactList.swift

```diff
@@ -0,0 +1,238 @@
+                    .default(Text("Switch User"), action: {
+                        self.modalPresented = ModalAction.switchUser
+                    }),
+                    .default(Text("Inspect DB"), action: {
+                        self.modalPresented = ModalAction.inspectDB
+                    }),
+                    .default(Text("Cancel")
+                )])
+            }.sheet(item: $modalPresented) { creationType in
+                if creationType == ModalAction.inspectDB {
+                    InspectorViewControllerWrapper(store: self.sObjectDataManager.store)
+                } else if creationType == ModalAction.switchUser {
+                    SalesforceUserManagementViewControllerWrapper()
+                }
+            }
+        }.alert(isPresented: $logoutAlertPresented, content: {
+            Alert(title: Text("Are you sure you want to log out?"),
+                  primaryButton: .destructive(Text("Logout"), action: {
+                      UserAccountManager.shared.logout()
+                  }),
+                  secondaryButton: .cancel())
+        })
+    }
+}
+
+struct ContactCell: View {
+    var contact: ContactSObjectData
+
+    var body: some View {
+        HStack {
+            Image(uiImage: ContactHelper.initialsImage(ContactHelper.colorFromContact(contact), initials: ContactHelper.initialsStringFromContact(contact))!)
+            VStack(alignment: .leading) {
+                Text(ContactHelper.nameStringFromContact(contact)).font(.appRegularFont(16))
+                Text(ContactHelper.titleStringFromContact(contact)).font(.appRegularFont(12)).foregroundColor(.secondaryLabelText)
+            }
+            Spacer()
+            if SObjectDataManager.dataLocallyUpdated(contact) {
+                Image(systemName: "arrow.2.circlepath").foregroundColor(.appBlue)
+            }
+            if SObjectDataManager.dataLocallyCreated(contact) {
+                Image(systemName: "plus").foregroundColor(.appBlue)
+            }
+        }
+        .padding([.all], 10)
+    }
+}
+
+struct SearchBar: UIViewRepresentable {
+    @Binding var text: String
+
+    class Coordinator: NSObject, UISearchBarDelegate {
+        @Binding var text: String
+
+        init(text: Binding<String>) {
+            _text = text
+        }
+
+        func searchBar(_ searchBar: UISearchBar, textDidChange searchText: String) {
+            text = searchText
+        }
+    }
+
+    func makeCoordinator() -> Coordinator {
+        return Coordinator(text: $text)
+    }
+
+    func makeUIView(context: UIViewRepresentableContext<SearchBar>) -> UISearchBar {
+        let searchBar = UISearchBar()
+        searchBar.placeholder = "Search"
+        searchBar.delegate = context.coordinator
+        return searchBar
+    }
+
+    func updateUIView(_ uiView: UISearchBar, context: UIViewRepresentableContext<SearchBar>) {
+        uiView.text = text
+    }
+}
+
+struct InspectorViewControllerWrapper: UIViewControllerRepresentable {
```

Comment:
Since these are generic wrappers to use the view controllers in SwiftUI, I’m wondering if we should add them to the SDK for convenience